autogpt_platform/backend/backend/data/execution.py

```diff
@@ -69,10 +69,55 @@ class GraphExecutionMeta(BaseDbModel):
     ended_at: datetime
 
     class Stats(BaseModel):
-        cost: int = Field(..., description="Execution cost (cents)")
-        duration: float = Field(..., description="Seconds from start to end of run")
-        node_exec_time: float = Field(..., description="Seconds of total node runtime")
-        node_exec_count: int = Field(..., description="Number of node executions")
+        model_config = ConfigDict(
+            extra="allow",
+            arbitrary_types_allowed=True,
+        )
+
+        cost: int = Field(
+            default=0,
+            description="Execution cost (cents)",
+        )
+        duration: float = Field(
+            default=0,
+            description="Seconds from start to end of run",
+        )
+        duration_cpu: float = Field(
+            default=0,
+            description="CPU sec of duration",
+        )
```

Comment:
Does this include the `node_exec_time_cpu`?

Maybe rename this `cpu_time`, as otherwise it gets confusing with `duration`+`duration_cpu` (to me)?

```suggestion
        cpu_time: float = Field(
            default=0,
            description="Seconds of CPU time used by the run",
        )
```

Reply:
duration_cpu here is `duration` but instead of walltime it's CPU time.
node_exec_time_cpu is `node_exec` but instead of walltime it's CPU time.

Hmm you are not supposed to do `duration+duration_cpu` it's not related.

Reply:
I've changed them to 

duration_cpu_only
node_exec_time_cpu_only